### DISCUSSION TITLE
How to prevent the use of the actual path when opening a symlink file

### DISCUSSION THREAD

**Discussion by SunJun8** [nvim-tree/nvim-tree.lua#2308] [Q&A]
Is there an option to disable the behavior of automatically opening the actual file path when opening a symlink?

**Comment by gegoune** [comment]
Out of interest. What would you like to happen instead?

**Reply by SunJun8** [reply]
I typically focus on C development and frequently use bear to produce compilation databases. The project could include symlink, which can cause issues with lsp when opening the original file rather than the symlink file in Vim. I'm uncertain if this requirement is a feature of nvim-tree or of Vim itself.

**Reply by alex-courtis** [reply]
neovim will open a buffer with the symlink, wheras nvim-tree will open a buffer with the absolute path.

We could attempt to do the same as neovim. We would, however, need to alter the behaviour of find file etc. to match. It's not quite as simple to implement as it might seem.
